typescript/packages/subsurface-viewer/src/components/Map.tsx

```diff
@@ -896,8 +896,25 @@ const Map: React.FC<MapProps> = ({
                     style={scale.cssStyle ?? {}}
                 />
             ) : null}
-            <StatusIndicator layers={deckGLLayers} isLoaded={isLoaded} />
-            {coords?.visible ? <InfoCard pickInfos={hoverInfo} /> : null}
+            {!isRenderedCallback && !isLoaded && (
+                <div
+                    style={{
+                        alignItems: "center",
+                        display: "flex",
+                        justifyContent: "center",
+                        position: "absolute",
+                        height: "90%",
+                        width: "90%",
+                        zIndex: 200,
+                    }}
+                >
+                    <StatusIndicator
+                        layers={deckGLLayers}
+                        isLoaded={isLoaded}
+                    />
+                </div>
```

Comment:
I think centering the progress indicator is too obtrusive. Maybe it could be repositioned to the lower corner.

Moreover, I think we can remove the whole `ProgressIndicator` and instead provide a progress callback similar to (or replacing) the `isRenderedCallback`, ie. something like `loadProgress(progress: number)`, where `progress` is in [0..1]. That way the consumer of this component could add use the `ProgressIndicator` as a `ViewAnnotation` or use any progress display any way they want.

Reply:
OK to move it to lower corner.

Unfortunately, there is no way to have a `progress` number. Thus the `isRenderedCallback` is indeed the way to provide the only available information (`on` or `off`) that allows the client code to display this information...

Reply:
change callback to `onRenderedProgress` which takes a number between 0 and 100.
moved default feedback to bottom right:
![image](https://github.com/equinor/webviz-subsurface-components/assets/93762994/9839ff49-da4d-4e62-b38d-3e95ed673e0c)

Example with use of callback:
![image](https://github.com/equinor/webviz-subsurface-components/assets/93762994/0b2e5416-74e0-4ed7-8ea7-59dd43afd3b0)